pandas/core/indexes/datetimelike.py

```diff
@@ -78,7 +78,7 @@ class DatetimeIndexOpsMixin(ExtensionOpsMixin):
     common ops mixin to support a unified interface datetimelike Index
     """
 
-    _data = None
```

Comment:
Were these solely set to `None` before for annotation purposes? I think need to be careful in changing this if the intent was solely solely for that purpose

Reply:
these were here pre-annotations, so I think it was so that the attribute always exists